lib/atom.fnl

```diff
@@ -0,0 +1,47 @@
+(fn atom
```

Comment:
Docstring, if we add something like (taken from clojure documentation):

```
;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
;; Atoms provide a way to manage shared, synchronous, independent state. ;;
;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;

```
will that accurately reflect it?

Reply:
Done in #36